_includes/_masthead.html

```diff
@@ -49,13 +51,15 @@
 {% elsif page.header.image_fullwidth %}
 
 <div id="masthead">
-	<div class="row">
-		<div class="small-12 columns">
-			<a id="logo" href="{{ site.url }}" title="{{ site.title }} – {{ site.slogan }}">
-				<img src="{{ site.url }}{{ site.baseurl }}/assets/img/{{ site.logo }}" alt="{{ site.title }} – {{ site.slogan }}">
-			</a>
-		</div><!-- /.small-12.columns -->
-	</div><!-- /.row -->
+{% if site.logo %}
+  <div class="row">
```

Comment:
I think indentation should be consistent with the rest of the file/project.

Reply:
As you like.  And just for fun, [hard tabs or spaces](https://en.wikipedia.org/wiki/Indentation_style#Tabs.2C_spaces.2C_and_size_of_indent)

Reply:
[I prefer tabs](https://youtu.be/SsoOG6ZeyUI) at home (personalized tab size by the code's reader, easier navigation in editor, distinguished indentation vs alignment whitespace), but don't mind any size spaces or tabs as long as it's consistent in the project/company.